docs/source/kubernetes/gettingstarted.rst

```diff
@@ -0,0 +1,112 @@
+.. _kubernetes_gettingstarted:
+
+Getting Started (Alpha)
+=======================
+
+The Kubernetes Provider (Alpha) is an optional package which an be installed to enable writing
+policies which interact with Kubernetes related resources.
+
+
+.. kubernetes_install-cc:
+
+Install Kubernetes Plugin
+-------------------------
+
+First, ensure you have :ref:`installed the base Cloud Custodian application
+<install-cc>`. Cloud Custodian is a Python application and must run on an
+`actively supported <https://devguide.python.org/#status-of-python-branches>`_
+version. 
+
+Once the base install is complete, you are now ready to install the Kubernetes provider package
+using one of the following options:
+
+Option 1: Install released packages to local Python Environment
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: bash
+
+    pip install c7n
+    pip install c7n_kube
+
+
+Option 2: Install latest from the repository
+"""""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: bash
+
+    git clone https://github.com/cloud-custodian/cloud-custodian.git
+    pip install -e ./cloud-custodian
```

Comment:
Should this be `cd tools/c7n_kube && poetry install` to follow along with our recommendation of using `poetry` in other places? 

Like: https://cloudcustodian.io/docs/developer/installing.html#installing-custodian